src/tests/cli_tests.py

```diff
@@ -1272,6 +1273,8 @@ def test_export_revocation(self):
         os.close(pipe)
         self.assertEqual(ret, 0)
         self.assertTrue(os.path.isfile(OUT_ALICE_REV))
+        with open(OUT_ALICE_REV, "rb") as armored:
```

Comment:
You may use helper function `file_text()` here for less code.

Reply:
Thanks, good to know.
But, I'm a bit concerned that file_test() definition does `replace('\r\r', '\r')`. Is that right?
Until this is clarified, I'll leave the code as it was.

Reply:
Not sure what it was aimed for. It came via this commit: https://github.com/rnpgp/rnp/commit/56d9ae9d258a8603eeeac7bb41e637020e0a0eb9

Probably some mistyping. Anyway, it worked for now )